### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/sentry/commit/036c5f2d9758c587288601f6c615f29a8ea0f4a1#commitcomment-76794577

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/sentry/commit/a4e27e9e8a4d00aa2aa160a72bb4d86aada1181b

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

I will be annotating this PR, and I fully expect to be asked to drop certain files.

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

**Comment by untitaker** [comment]
@jsoref I apprechiate the effort, but you just cc'ed a ton of teams in the same PR. I would recommend to create multiple PRs partitioned by some approximation of code module + whether the change is in a code comment or not. It's not possible to safely review and deploy a change like this.

File: tests/snuba/api/endpoints/test_group_details.py [hunk 1]
```diff
@@ -166,7 +166,7 @@ def test_assigned_to_unknown(self):
         )
         assert response.status_code == 200
         response = self.client.put(
-            url, {"assignedTo": "user@doesntexist.com", "status": "unresolved"}, format="json"
+            url, {"assignedTo": "user@doesnotexist.com", "status": "unresolved"}, format="json"
```
Comment (jsoref) [inline tests/snuba/api/endpoints/test_group_details.py:169]: In principle, it's bad form to use domains you don't control, sometimes I even include changes to use example.com/example.org/friends (there are a set of reserved domains -- https://datatracker.ietf.org/doc/html/rfc6761), but I didn't for this PR.

File: tests/snuba/api/endpoints/test_organization_stats_v2.py [hunk 2]
```diff
@@ -184,14 +184,14 @@ def test_unknown_groupby(self):
         response = self.do_request(
             {
                 "field": ["sum(quantity)"],
-                "groupBy": ["cattygory"],
+                "groupBy": ["category_"],
```
Comment (jsoref) [inline tests/snuba/api/endpoints/test_organization_stats_v2.py:187]: I'm assuming it doesn't matter as long as it isn't `category`, and hence offering an alternative.

**Review by jsoref — COMMENTED** [review]
Most corrections were automatically suggested by Google Sheets, but all fault is mine.

**Comment by jsoref** [comment]
@untitaker: Sorry, the PR template didn't warn me that this would happen. I'm happy to do a split, although it won't happen this week, and this repo has moved quite quickly (I had to rebase a couple of times which surprised me).

I'm uncertain how closely coupled/loosely coupled the different directories are. If someone could provide at least an outline of directory splits, that'd help.

File: static/app/components/replays/player/scrubber.tsx [hunk 3]
```diff
@@ -93,7 +93,7 @@ const Wrapper = styled('div')`
   }
 `;
 
-export const TimelineScubber = styled(Scrubber)`
+export const TimelineScrubber = styled(Scrubber)`
```
[the remark this answers is not in the document]
Comment (ryan953) [reply static/app/components/replays/player/scrubber.tsx:96]: It's only used in replayTimeline.tsx which is also updated. 👍

**Review by ryan953 — APPROVED** [review]
Looked at files with `replay` or `prof` in the name for emerging tech, looking good there.

**Review by wedamija — CHANGES_REQUESTED** [review]
Just reiterating what @untitaker said above - this is too dangerous for us to merge as is.

I recommend splitting all javascript and python changes into separate prs.

Within that, probably a pr per module will make this more reviewable.

**Comment by jsoref** [comment]
I'm going to stop splitting for the time being. Once a bunch of the split PRs are merged, I'll rebase the remainder and see about splitting again.

(Unless someone wants to suggest a specific module that's interested in eagerly entertaining a PR.)

**Comment by armenzg** [comment]
@jsoref thanks for this.

For now let's close it and open individual PRs for specific teams (I'm mainly concerned of removing the review request from the queue since we can't review in its current shape).

**Pull request closed without merging**